tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -208,3 +208,33 @@ Create disables VC destroy
     Should Be Equal As Integers  ${rc}  0
     Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Contain  ${output}  Destroy_Task
     Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Not Contain  ${output}  Destroy_Task
+
+Parallel creates with same container name
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull ${busybox}
+    Should Be Equal As Integers  ${rc}  0
+
+    ${suffix}=  Evaluate  '%{DRONE_BUILD_NUMBER}-' + str(random.randint(1000,9999))  modules=random
```

Comment:
This still doesn't avoid typo issues. I think the following would be more maintenance friendly:
```
${suffix}=  Generate Random String  8
Set Test Variable  $container_name  testDuplicates-${suffix}

```

Reply:
Fixed!